services/federator/test/unit/Test/Federator/Validation.hs

```diff
@@ -36,6 +38,11 @@ import Test.Polysemy.Mock (evalMock)
 import Test.Tasty
 import Test.Tasty.HUnit
 import Wire.API.Federation.GRPC.Types
+import Wire.Network.DNS.SRV (SrvTarget (..))
+
+mockDiscoveryTrivial :: Sem (DiscoverFederator ': r) x -> Sem r x
+mockDiscoveryTrivial = Polysemy.interpret $ \(DiscoverFederator dom) ->
+  pure . Right $ SrvTarget (Text.encodeUtf8 (domainText dom)) 443
```

Comment:
Given that this function is copy-pasted in `ExternalServer.hs`, how about moving it to a `Util` or `Interpret` module?

Reply:
I've now imported it from the other module directly.